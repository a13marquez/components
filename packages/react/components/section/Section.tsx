import * as React from 'react';
import { StyledProductsSection } from '../styled/sections/StyledProductsSection';

export const Section: React.FunctionComponent =  (props) => (
  <StyledProductsSection>
    {props.children}
  </StyledProductsSection>
)
