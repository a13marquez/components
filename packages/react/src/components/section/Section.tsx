import * as React from 'react';
import StyledSectionWithPadding 
  from '@bit/a13marquez.styled-components.styled-padded-section';

export const Section: React.FunctionComponent =  (props) => (
  <StyledSectionWithPadding>
    {props.children}
  </StyledSectionWithPadding>
)
