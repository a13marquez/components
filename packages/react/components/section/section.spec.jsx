import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import { Section } from './Section';

describe('Section Component: ', () => {
  let section;
  beforeEach(() => {
    section = mount(<Section>
      <div />
    </Section>);
  });
  afterEach(() => {
    section = null;
  });
  test('It renders Section component', () => {
    expect(section).toMatchSnapshot();
  });

  test('It renders Section with props', () => {
    expect(section.props().children.type).toBe('div');
  });
});
