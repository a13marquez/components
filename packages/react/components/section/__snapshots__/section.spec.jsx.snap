// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Component:  It renders Section component 1`] = `ReactWrapper {}`;
